Sources/TuistDependencies/SwiftPackageManager/Utils/SettingsMapper.swift

```diff
@@ -143,7 +143,7 @@ extension PackageInfo.Target.TargetBuildSettingDescription.Setting {
 
 extension PackageInfo.Platform {
     fileprivate func graphPlatform() throws -> TuistGraph.Platform {
-        switch platformName {
+        switch platformName.lowercased() {
```

Comment:
There are 2 more places that we use `platformName` where case is important, both are in `PackgeInfoMapper.swift`.  It would be nice if we could unify the lowercasing to a single property on `PackageInfo.Platform` to prevent diverging implementations in other areas that use `platformName`

Reply:
I've tried but I need to import TuistGraph and then I have a circular dependency warning.